pyphare/pyphare/pharesee/particles.py

```diff
@@ -136,7 +137,7 @@ def split(self, sim): # REQUIRES C++ PYBIND PHARE LIB
           weights=split_pyarrays[2],
           charges=split_pyarrays[3],
           v=np.asarray(split_pyarrays[4]).reshape(int(len(split_pyarrays[4]) / 3), 3),
-          dl = self.dl/2
+          dl = self.dl[0]*0.5 + np.zeros((split_pyarrays[2].size,self.ndim))
```

Comment:
may we use an explicit var for the origin of the 0.5 factor : inverse of refinement ratio I guess